doc/source/tutorials/local/Exploring_data/Retrieving_remote_data.rst

```diff
@@ -0,0 +1,215 @@
+
+.. code:: ipython3
+
+    wao_data_level1.plot_timeseries(title="WAO - Level 1 data")
+
+Forcing retrieval
+~~~~~~~~~~~~~~~~~
+
+As ICOS data is cached by OpenGHG you may sometimes need to force a
+retrieval from the ICOS Carbon Portal.
+
+If you retrieve data using ``retrieve_icos`` and notice that it does not
+return the most up to date data (compare the dates with those on the
+portal) you can force a retrieval using ``force_retrieval``.
+
+.. code:: ipython3
+
+    new_data = retrieve_atmospheric(site="WAO", species="CH4", data_level=1, force_retrieval=True)
+
+Here you may notice we get a message telling us there is no new data to
+process, if you force a retrieval and there is no newer data you'll see
+this message.
+
+CEDA
+----
+
+To retrieve data from CEDA you can use the ``retrieve_ceda`` function
+from ``openghg.client``. This lets you pull down data from CEDA, process
+it and store it in the object store. Once the data has been stored
+successive calls will retrieve the data from the object store.
+
+   **NOTE:** For the moment only surface observations can be retrieved
+   and it is expected that these are already in a NetCDF file. If you
+   find a file that can't be processed by the function please `open an
+   issue on
+   GitHub <https://github.com/openghg/openghg/issues/new/choose>`__ and
+   we'll do our best to add support that file type.
+
+To pull data from CEDA you'll first need to find the URL of the data. To
+do this use the `CEDA data browser <https://data.ceda.ac.uk/badc>`__ and
+copy the link to the file (right click on the download button and click
+copy link / copy link address). You can then pass that URL to
+``retrieve_ceda``, it will then download the data, do some
+standardisation and checks and store it in the object store.
+
+We don't currently support downloading restricted data that requires a
+login to access. If you'd find this useful please open an issue at the
+link given above.
+
+Now we're ready to retrieve the data.
+
+.. code:: ipython3
+
+    from openghg.retrieve.ceda import retrieve_surface
+
+.. code:: ipython3
+
+    url = "https://dap.ceda.ac.uk/badc/gauge/data/tower/heathfield/co2/100m/bristol-crds_heathfield_20130101_co2-100m.nc?download=1"
+
+.. code:: ipython3
+
+    hfd_data = retrieve_surface(url=url)
+
+Now we've got the data, we can use it as any other ``ObsData`` object,
+using ``data`` and ``metadata``.
+
+.. code:: ipython3
+
+    hfd_data.plot_timeseries()
+
+Retrieving a second time
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The second time we (or another use) retrieves the data it will be pulled
+from the object store, this should be faster than retrieving from CEDA.
+To get the same data again use the ``site``, ``species`` and ``inlet``
+arguments.
+
+.. code:: ipython3
+
+    hfd_data2 = retrieve_surface(site="hfd", species="co2")
+
+.. code:: ipython3
+
+    hfd_data2
+
+Cleanup the temporary object store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: ipython3
+
+    tmp_dir.cleanup()
```

Comment:
*(Same comment as above)* Does clear_tutorial_store() need to be added here (at least for consistency)?